src/illuminator/builder/model.py

```diff
@@ -156,12 +167,15 @@ def init(self, sid, time_resolution, **sim_params):  # can be use to update mode
         # self.model = self.load_model_class(self.model_data['model_path'], self.model_data['model_type'])
         return self._model.simulator_meta
     
-    def create(self, num: int) -> List:
+    def create(self, num:int, model:str, **model_params) -> List[dict]: # This change is mandatory. It MUST contain num and model as parameters otherwise it receives an incorrect number of parameters
```

Comment:
As the comment states, we must have the second parameter, `model` otherwise we always call `n+1` number of parameters, instead of `n` number of parameters. 

This is due to the way `mosaik` itself calls this function.